code/modules/reagents/chemistry/machinery/chem_mass_spec.dm

```diff
@@ -30,35 +21,22 @@ This will not clean any inverted reagents. Inverted reagents will still be corre
 	var/list/log = list()
 	///Input reagents container
 	var/obj/item/reagent_containers/beaker1
-	///Output reagents container
-	var/obj/item/reagent_containers/beaker2
-	///multiplies the final time needed to process the chems depending on the laser stock part
-	var/cms_coefficient = 1
 
 /obj/machinery/chem_mass_spec/Initialize(mapload)
 	. = ..()
 	ADD_TRAIT(src, DO_NOT_SPLASH, src.type)
-	if(mapload)
-		beaker2 = new /obj/item/reagent_containers/glass/beaker/large(src)
+
+/obj/machinery/chem_mass_spec/examine()
+	. += span_notice("Left-Click on the machine with a beaker to insert. Alt-Left-Click to remove.")
```

Comment:
Is this a new examine text?

If so it should be contextual screentips instead.

Reply:
Could you elaborate on this?

Reply:
If you added this, then it shouldn't be (just) in examine:

https://github.com/tgstation/tgstation/blob/master/code/datums/elements/screentips/README.md

Reply:
They are the text you see on your screen when you hover over something

Reply:
I changed the description back because it has a use again, as well as gave it contextual screentips for how to add beakers